evaluation_experiments/EXP3.py

```diff
@@ -0,0 +1,238 @@
+#The following script runs the notebook specified in in_nb with 
```

Comment:
It's not clear what EXP1, 2, and 3 are. For future readers it would be useful to include some info on the experiments in the header of each file.

Reply:
added a header with description on each file.